### DISCUSSION TITLE
Dynamically match NVIM theme with HYDE theme?

### DISCUSSION THREAD

**Discussion by Nic-Vladimir** [prasanthrangan/hyprdots#1694] [Q&A]
Has anybody made a script to change the NVIM theme or at least the background color when the HYDE theme changes?

The tokyonight color scheme is bugging me in NVIM while using any other theme than tokyonight. The terminal matches the theme just right but not nvim as you can see in the pic.
![image](https://github.com/user-attachments/assets/698709d9-e11a-420c-b1b3-df3f635ecfcf)

**Comment by kRHYME7** [comment]
Checkout the scattered discussions about it in this repo
